packages/components/src/components/vertical-tabs/vertical-tab-content-container/vertical-tab-content-container.jsx

```diff
@@ -26,24 +25,32 @@ class VerticalTabContentContainer extends React.PureComponent {
                     >
                         {
                             action_bar.map(({ component, icon, onClick }, idx) => {
-                                const Component = component;
-                                return (
-                                    component ?
+                                if (component) {
+                                    const Component = component;
+                                    return (
                                         <Component key={idx} />
-                                        :
-                                        <div
-                                            id={`dt_${id}_close_icon`}
-                                            className='vertical-tab__action-bar-wrapper'
-                                            key={idx}
-                                            onClick={onClick}
-                                        >
-                                            <Icon
-                                                className='vertical-tab__action-bar--icon'
-                                                icon={icon}
-                                            />
-                                        </div>
+                                    );
+                                }
+                                const Icon = React.cloneElement(icon, {
+                                    className: classNames(
+                                        'vertical-tab__action-bar--icon', {
+                                            [icon.props.className]: !!icon.props.className,
+                                        }),
+                                });
+
+                                return (
+                                    <div
+                                        id={`dt_${id}_close_icon`}
+                                        className='vertical-tab__action-bar-wrapper'
+                                        key={idx}
+                                        onClick={onClick}
+                                    >
+                                        { Icon }
```

Comment:
Question: Is this `<Icon />` possible to use here?

Reply:
Resolving cause outdated.